guidelines/1.3.2.md

```diff
@@ -14,6 +14,18 @@ When the order of content is important, the content order must be preserved no m
 *   Tabular data is displayed using CSS to present the appearance of a table, but the proper HTML markup has not been used;
 *   The tabindex attribute has been used to force a tab order through the content.
 
+##### Code examples
+
+###### iOS
+
+```swift
+// Setting a custom order for child elements
+containerView.isAccessibilityElement = false
```

Comment:
This isn't a typo, the containerView needs to have `isAccessibilityElement` set to `false`:

>Note, however, that the container view itself is not an accessibility element because users interact with the contents, not with the container. This means that a container view that implements the UIAccessibilityContainer methods must set to false the isAccessibilityElement property of the UIAccessibility informal protocol.

https://developer.apple.com/documentation/uikit/accessibility/uiaccessibilitycontainer